travis/update-release.py

```diff
@@ -158,18 +168,27 @@ def check_rollback_artifacts(r_stream,tag):
     for image in z_container_images:
```

Comment:
could we collapse the 2 for loops, and instead check if rollback is required in the same loop, this way we could also delete the `get_container_image` data.
Sorry for the nit, that function was supposed to be consumed for the rollback functionality anyways and is only called if this loop fails to find a mismatch hence it most likely will not even be called in our releases.
```
if quaySha != image["quay"][0]["sha"] or dockerSha != image["docker"][0]["sha"]:
   run_git_commands(image_id, image)
else
   copyfile("/tmp/" + GIT_LOCAL_DIR + "/docs/release_artifacts/" + RELEASE_TAG + "/z/" + image["name"], "/tmp/" + GIT_LOCAL_DIR + "/docs/release_artifacts/" + RELEASE_TAG + DIR + image["name"])
```

Reply:
Sure addressed